scripts/retire.py

```diff
@@ -35,8 +36,16 @@ def move_file(filename):  # pragma: no cover
     os.renames(filename, new_filename)
 
 
+def validate_end_date(ctx, value):
+    try:
+        end_date = datetime.strptime(value, '%Y-%m-%d')
+        return end_date
```

Comment:
since this returns a datetime instead of the original string I believe ( haven't had a chance to check yet) that this will show up differently in the actual file, should we just return the original value?